core/domain/acl_decorators_test.py

```diff
@@ -218,9 +218,18 @@ def setUp(self):
         rights_manager.publish_exploration(self.owner, self.published_exp_id)
         rights_manager.publish_collection(self.owner, self.published_col_id)
 
-    def test_guest_is_redirected_to_login_page(self):
-        response = self.mock_testapp.get(
-            '/mock/%s' % self.published_col_id, expect_errors=True)
+    def test_guest_cannot_edit_collection_on_JSON_return_type(self):
```

Comment:
Let's use "via_json_handler", "when_using_html_handler" or similar, instead of "on_json_return_type", "on_html_return_type". That's more semantically correct (I think that "on" is not the correct preposition here).

Ditto below and elsewhere.

Reply:
fixed.